recipes/cspice/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "cspice" %}
+{% set version = "66" %}
+{% set root_url = "https://naif.jpl.nasa.gov/pub/naif/misc/toolkit_N%(version)04d/C"|format(version=version | int) %}
+
+{% set platform_segment = "PC_Linux_GCC_64bit" %}                                      # [linux64]
+{% set sha256 = "93cd4fbce5818f8b7fecf3914c5756b8d41fd5bdaaeac1f4037b5a5410bc4768" %}  # [linux64]
+{% set platform_segment = "PC_Windows_VisualC_32bit" %}                                # [win32]
+{% set sha256 = "db48e5beddc0e50f7a0224bb2c6f2b39fc38f84628ad5cd455e847fc70aa5e75" %}  # [win32]
+{% set platform_segment = "PC_Windows_VisualC_64bit" %}                                # [win64]
+{% set sha256 = "b29dfb0b41ae8322c60ee9d162beb97433d21e12843ecbe03d819e5edef45920" %}  # [win64]
+{% set platform_segment = "MacIntel_OSX_AppleC_64bit" %}                               # [osx]
+{% set sha256 = "f5d48c4b0d558c5d71e8bf6fcdf135b0943210c1ff91f8191dfc447419a6b12e" %}  # [osx]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}.tar.Z  # [not win]
+  fn: {{ name }}.zip    # [win]
+  url: {{ root_url }}/{{ platform_segment }}/packages/{{ name }}.tar.Z  # [not win]
+  url: {{ root_url }}/{{ platform_segment }}/packages/{{ name }}.zip    # [win]
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  features:
+    - vc9    # [win and py27]
+    - vc10   # [win and py34]
+    - vc14   # [win and py35]
+    - vc14   # [win and py36]
+
+requirements:
+  build:
+    - gcc     # [not win]
```

Comment:
Also why do we need `gcc` on macOS?

Reply:
Because it's needed to build a shared module.